tests/test-customize-changesets-controller.php

```diff
@@ -1384,55 +1299,41 @@ public function test_update_item_edit_not_future_date_with_past_date() {
 		) );
 		$response = $this->server->dispatch( $request );
 
-		$this->assertErrorResponse( 'not_future_date', $response );
+		$this->assertErrorResponse( 'rest_invalid_param', $response );
 		$this->assertSame( $date_before, get_post( $manager->changeset_post_id() )->post_date_gmt );
 	}
 
 	/**
 	 * Test that update_item() rejects scheduling a changeset when it has a past date.
+	 *
+	 * @todo Verify that this is true -- not clear why should be be rejected.
```

Comment:
That makes sense, my bad, misunderstood this unit tests and initially thought that it would change the date and the status both, not just the status.